lib/src/clojure_lsp/feature/code_actions.clj

```diff
@@ -318,182 +111,159 @@
          {:title      (format "Add require '[%s %s %s]'" ns (if alias ":as" ":refer") (or alias (str "[" refer "]")))
           :kind       :quick-fix
           :preferred? true
-          :data       {:id        "add-missing-require"
-                       :uri       uri
-                       :line      (:line position)
-                       :character (:character position)}})
+          :command    {:title     "Add missing require"
+                       :command   "add-missing-require"
```

Comment:
This one is not working, it's tricky, the correct should be `add-missing-libspec` for legacy reasons

Reply:
Good catch. I thought I tested that, but I must have been testing `add-require-suggestion`, which looks the same in the action menu. Two questions...

1. What's the difference between a "missing require" and a "require suggestion"?
2. Originally, resolve-action called `add-missing-libspec` with `:version 0`. Do you know why and whether that's necessary to maintain?

Reply:
1. Clojure-lsp used to have only `add-missing-libspec`, which would add a require that already exists on the project somewhere, the `add-require-suggestion` as the name say suggest some namespaces know via kondo analysis, the code actions are similar to make it more user friendly, and that's why I always suggest relying on the code actions than manual commands.

2. the version is something we use to track how much times the file was changed, via user or lsp, this helps know if the file we are changing now has a version outdated during some async stuff, but for code actions this usually is not a issue, I think any number here would work

Reply:
OK, fixed. Thanks again for catching that.

Reply:
I've been thinking about this... Each missing-require action calculates what the alias/refer should be but then throws that away, forcing the server to recalculate it when the action is selected. If those actions also used the `add-require-suggestion` command, which includes the alias/refer in the arguments, we could avoid that recalculation. It'd be a small performance improvement, but maybe nice anyway. We probably couldn't remove the `add-missing-libspec` command entirely—I'm sure some editor or user calls it directly—but it'd become an infrequently used code path.

Reply:
I don't think we should bother with performance during code action execution unless a huge performance issue, but if you have a way to make all add-require logic in a single command, sounds a good idea.
I know a lot of vim users use that command, so less breaking changes would be better